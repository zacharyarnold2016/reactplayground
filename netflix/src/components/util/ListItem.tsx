const ListItem = (props: any) => {
  const value = props.value;
  return (
    // Wrong! There is no need to specify the key here:
    <li>{value}</li>
  );
};

export default ListItem;
