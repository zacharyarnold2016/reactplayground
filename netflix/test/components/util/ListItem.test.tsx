import { expect } from "@jest/globals";
import renderer from "react-test-renderer";
import ListItem from "../../../src/components/util/ListItem";

describe("ListItem", () => {
  test("renders", () => {
    const component = renderer.create(<ListItem value="test" />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
