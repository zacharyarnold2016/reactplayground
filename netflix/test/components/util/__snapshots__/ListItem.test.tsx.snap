// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItem renders 1`] = `
<li>
  test
</li>
`;
